packages/runtime/runtime-utils/src/index.ts

```diff
@@ -6,6 +6,7 @@
 export * from "./dataStoreHandleContextUtils";
 export * from "./dataStoreHelpers";
 export * from "./garbageCollectionUtils";
+export * from "./garbageCollector";
```

Comment:
this doesn't really seem like a util to me. actually most of these don't see like utils. to me utils should be things used by the container runtime, but also might be used independently, like by data stores.

Reply:
The garbageCollector is independent of container runtime or any other runtime concepts. The idea I had in mind is that any one who wishes to run GC on its data can use this garbage collector
 For example, a collection data store can use it to run GC to delete unused instances. Any data store or blob manager can also use it.

Reply:
should it just be its own package then? i just don't want utils to be a dumping ground which is what frequently happens with utils packages

Reply:
no need to block on it. just something to consider

Reply:
I moved it to a new package under packages/utils. Please let me know if this looks good.

Reply:
I had to move IGraphNode to the new utils as well because of layer violations and cycling dependencies.